src/core/stdc/config.d

```diff
@@ -21,40 +69,10 @@ else version (TVOS)
 else version (WatchOS)
     version = Darwin;
 
-extern (C):
-@trusted: // Types only.
-nothrow:
-@nogc:
-
 version( Windows )
 {
-    struct __c_long
-    {
-      pure nothrow @nogc @safe:
-        this(int x) { lng = x; }
-        int lng;
-        alias lng this;
-    }
-
-    struct __c_ulong
-    {
-      pure nothrow @nogc @safe:
-        this(uint x) { lng = x; }
-        uint lng;
-        alias lng this;
-    }
-
-    /*
-     * This is cpp_long instead of c_long because:
-     * 1. Implicit casting of an int to __c_long doesn't happen, because D doesn't
-     *    allow constructor calls in implicit conversions.
-     * 2. long lng;
-     *    cast(__c_long)lng;
-     *    does not work because lng has to be implicitly cast to an int in the constructor,
-     *    and since that truncates it is not done.
-     * Both of these break existing code, so until we find a resolution the types are named
-     * cpp_xxxx.
-     */
+    enum __c_long  : int;
```

Comment:
I had no clue that it was even legal to declare an `enum` with no members. That seems very weird. Is the point of doing this instead of using an `alias` so that it's a distinct type from `int`? I guess that that would mean that this technique is a `typedef` of sorts.

Reply:
> Is the point of doing this instead of using an alias so that it's a distinct type from int?

Yes, so that the compiler can do special mangling for that type. Although I don't see how it's any better than a struct. I proposed to add support for adding `pragma(mangle)` to alias declarations but Walter doesn't seem to like that.

Reply:
@jmdavis BTW, one use case of declaring an enum without any members is to use it as a UDA that doesn't contain any data.